src/components/popper/popper.ts

```diff
@@ -117,9 +117,11 @@ export class MPopper extends ModulVue implements PortalMixinImpl {
     }
 
     public update(): void {
-        if (this.popper !== undefined) {
-            this.popper.update();
-        }
+        this.$nextTick(() => {
+            if (this.popper !== undefined) {
+                this.popper.update();
```

Comment:
Je serais curieux de voir si on peut pas utiliser Popper.scheduleUpdate() à la place... 
Schedule an update, it will run on the next UI update available

Ça a l'air de faire pas mal ce qu'on veut :)

Reply:
En effet, ça fonctionne. Je viens de faire le changement dans le code.